module/actor/actor-sheet.js

```diff
@@ -50,6 +50,9 @@ export class yzecoriolisActorSheet extends ActorSheet {
 
   /** @override */
   async getData(options) {
+    // Slowly migrate keyArt to img whenever an actor sheet is opened.
+    this.object.migrateKeyArt();
```

Comment:
Migrate when the actor sheet opens. We could remove if we do a proper migration.

Reply:
Ya I think a proper migration would be in order here. Just so that the actor-sheet.js only ever needs to deal with and understand the latest data model.